src/pages/myNotes/myNotes.jsx

```diff
@@ -18,31 +21,58 @@ const MyNotes = ({
   isEditMode,
   setIsEditMode,
   onUpdateNote,
+  onAddNote,
+  onDeleteNote,
+  onFilterByDate,
+  onFilterByTitle,
+  getAllUsersSuccess,
+  allUsersData,
+  dragService,
+  setCurrentNote,
+  reOrder,
 }) => {
   const classes = useStyles();
   return (
     <div className={classes.root}>
       <Grid container spacing={3}>
         <Grid item xs={12}>
-          <Paper className={classes.paper}>
-            <h1>My list of Notes</h1>
-          </Paper>
+          <h1>My list of Notes</h1>
+        </Grid>
+        <Grid item xs={12} className={classes.alignLeft}>
+          <Button
+            variant="contained"
+            className={classes.headerBtn}
+            onClick={onAddNote}
+          >
+            Add Note
+          </Button>
+          <FilterByDate onFilterByDate={onFilterByDate} />
+          <FilterByTitle onFilterByTitle={onFilterByTitle} />
         </Grid>
         <Grid item xs={4}>
-          <NotesList
-            notes={notes}
-            activeNoteId={activeNoteId}
-            setActiveNoteId={setActiveNoteId}
-            isEditMode={isEditMode}
-            setIsEditMode={setIsEditMode}
-          />
+          {notes &&
+            notes.pages.map((page, i) => (
```

Comment:
Are you sure that using the index as a key - a good idea? Replace it please!! And tell me why it's a BAD practice to do such way.